envoy/api/v2/core/health_check.proto

```diff
@@ -139,7 +153,7 @@ message HealthCheck {
   // standard health check interval that is defined.
   //
   // The default value for "no traffic interval" is 60 seconds.
-  google.protobuf.Duration no_traffic_interval = 12;
+  google.protobuf.Duration no_traffic_interval = 13;
```

Comment:
You can't renumber existing fields. Just make `extension_health_check` field 13 instead.

Reply:
Thanks for letting me know about this.